README.md

```diff
@@ -17,19 +17,46 @@
 
 ## API
 ### URL
-> `http://localhost:3001/rooms/:id/` Dynamically render rooms based on listing id.
+> `http://localhost:3001/gallery/:listingId/` Dynamically render gallery based on listing id.
+<!-- this is the url for the photo-gallery module -->
 
 ### POST
-> `/add/:id/` Add a photo in listing.
+> `/:listingId/upload` Add a photo into a specific listing.
+
+**Example**
+req body:
+  `{
+   photo_url: String
+   caption: String
+  }`
+
+res: 201
 
 ### GET
-> `/:id/:photoId` Retreive a photo in listing.
+> `/gallery/:listingId/photos` Retreive all photos in the gallery for a specific listing. (or do I want to get ALL of the photos)
+<!--get all method querying the listings table-->
+
+**Example**
+`/gallery/1/photos` will return photo gallery for listingId: 1.
+`/gallery/8/photos` will return photo gallery for listingId: 8.
```

Comment:
You show the response on the other requests. I would consider showing what these results actually look like.

Reply:
I'll update the README once I test out the requests!

Reply:
Perfectly fine!